cmd/device-hub-cli/cmd_create.go

```diff
@@ -29,6 +31,18 @@ var createCommand = &cobra.Command{
 				return err
 			}
 
+			params, err := hub.DescribeListener(v.Type)
+
+			if err != nil {
+				return err
+			}
+
+			err = describe.Validate(v.Configuration, params)
```

Comment:
Validate isn't completely implemented yet, I think it would be nice to add a comment here warning about it.